### DISCUSSION TITLE
Error: Exception in HostFunction: unordered_map::at: key not found

### DISCUSSION THREAD

**Discussion by kouamegerard** [realm/realm-js#6065] [Q&A]
Hi everyone,
I have an error that I do not understand, two days ago not this error and since yesterday I launch my app it crashes with this error
**Exception in HostFunction: unordered_map::at: key not found**

**Comment by cyril36** [comment]
I dont know if you have react-native-get-random-values installed but i had react-native-get-random-values@1.9.0 and i downgraded it to 1.8.0 and it works

Steps:

Remove the line for react-native-get-random-values from package.json
Remove package-lock.json
Remove node_modules
Run npm i to install the dependencies
Run npm i react-native-get-random-values@1.8.0

**Reply by ChrisTregaskis** [reply]
Worked for me cheers mate

**Reply by ldphuong20122003** [reply]
It works. Thank you so much
